xml_inscriptions/bib_only/MA.Bos.MFA.G.01.Res.08.34b.xml

```diff
@@ -62,14 +130,29 @@
       <revisionDesc>
          <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-06-23" who="Jean Wanlass">adding information to bib only stub</change>
       </revisionDesc>
    </teiHeader>
-   <!--facsimile element goes here-->
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc>photo: MFA Res.08.34b</desc>
+         <graphic url="MA.Bos.MFA.G.Res.08.34b.jpg"/>
+      </surface>
+   </facsimile>
+   
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+            <ab><supplied reason="lost">ΔI</supplied>EIYΨICTШΔAΦNIC <supplied reason="lost">EU</supplied>XHN ANEϴHKE</ab></div>
```

Comment:
Because this is understandable, we use interpretive (rather than just diplomatic) transcription here. So, you should have `<supplied reason="lost">`Δι`</supplied>`εὶ ὑψίστω Δάφνις `<supplied reason="lost">`εὐ`</supplied>`χὴν ἀνέθηκε.

We could also add some `<persName>` elements around the god and Daphnis here, if we wanted to enhance further. (Also, is a κατ' also missing? Usually that's the formula with εὐχήν.)